count_timer/old_demos/demo_thread.py

```diff
@@ -60,7 +60,6 @@ def count():
 def take_input():
     while counter.remaining > 0:
         with term.cbreak():
-            key = term.inkey()
```

Comment:
I assume this line was removed because the return variable, key, is not used elsewhere. This will break the code, however. The call to inkey() is needed to accept user input from the keyboard (to pause/resume the counter). I suppose we can just change it to `term.inkey()`.

Reply:
@jsh somehow this is still not resolved. I'll be damned if I can figure out GH's PR mechanism. Anyway, the way I do it on PRs in Bitbucket is to just make the change on the same branch locally where I originally pushed the code from, then add/commit that single change and push again. It should update the PR on GH.